README.md

```diff
@@ -19,8 +19,6 @@ linux system).
 
 The dependencies to build the images are:
   - Yocto poky or later
-  - python3.4 or later (Installed on host machine)
-  - Lib python3-jsonschema (Installed on host machine)
```

Comment:
Why removing this kind of documentation?

Even now yocto will check it, I would like to read this kind of documentation when I am having the first approach of a project.

Maybe, we could add more information about the dependencies ?

What do you think ?